### ISSUE TITLE
Missing FileSaver.min.js

### ISSUE THREAD

**Issue by jay8t6** [eligrey/FileSaver.js#436]
As of version 1.3.6, the filesaver.min.js file is missing, and its breaking the code. Can you please add it back?

**Comment by edlangjr** [comment]
Agreed.  This is due to NPM package not being created correctly. See issue #433
